src/components/GroceryList/GroceryList.jsx

```diff
@@ -79,11 +85,26 @@ function GroceryList() {
       });
       let iconChange = document.getElementById("saved");
       iconChange.src = bookmarkGreenCheck;
+      setSnapshot(items) //CHANGES FROM JON
       setUpdates(false);
     } else {
       return;
     }
   };
+
+  const handleUndo = () => {
+    setItems([])
+    axios.get("http://localhost:2996/")
```

Comment:
can this be done without making an API call? i think it would be more efficent and is still possible to do so without making an API call.